hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/parse/SqlAlterJob.java

```diff
@@ -124,7 +124,7 @@ public void validate(SqlValidator validator, SqlValidatorScope scope) {
 
             switch (key) {
                 case "snapshotIntervalMillis":
-                    deltaConfig.setSnapshotIntervalMillis(parseLong(validator, option));
+                    deltaConfig.setSnapshotIntervalMillis(QueryUtils.parseLong(validator, option));
```

Comment:
some changes to job options for ANALYZE job using `ALTER JOB` should be not supported (in line with ANALYZE requirements). I created separate point for that in HZ-2863 and this can be done separately